service/pixelated/resources/backup_account_resource.py

```diff
@@ -44,3 +47,19 @@ def render_GET(self, request):
     def _render_template(self, request):
         site = BackupAccountPage()
         return renderElement(request, site)
+
+    def render_POST(self, request):
+        account_recovery = AccountRecovery(self._authenticator.bonafide_sesssion)
+
+        def update_response(response):
+            request.setResponseCode(NO_CONTENT)
+            request.finish()
+
+        def error_response(response):
+            request.setResponseCode(INTERNAL_SERVER_ERROR)
+            request.finish()
+
+        d = account_recovery.update_recovery_code("123")
+        d.addCallbacks(update_response)
+        d.addErrback(error_response)
```

Comment:
It'd be better to simplify the callbacks chain:

```
d.addCallbacks(update_response, error_response)
```

Or, alternatively, remove the `s` from `addCallbacks`:
```
d.addCallback(update_response)
d.addErrback(error_response)
```